com.unity.multiplayer.mlapi/Tests/Runtime/NetworkUpdateLoopTests.cs

```diff
@@ -11,8 +11,45 @@ namespace MLAPI.RuntimeTests
 {
     public class NetworkUpdateLoopTests
     {
+        [UnityTest]
+        public IEnumerator RegisterAndUnregisterSystems()
+        {
+            // caching the current PlayerLoop (it will have NetworkUpdateLoop systems registered)
+            var cachedPlayerLoop = PlayerLoop.GetCurrentPlayerLoop();
+            {
```

Comment:
```suggestion
```

You don't do this in any other tests, and I think if you really want to ensure you leave things "as they were" you'd maybe want to trigger a domain reload instead.